ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventLRspec.json

```diff
@@ -343,5 +343,24 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CLAIMANT_RESPONSE_SPEC",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[APPLICANTSOLICITORONE]"
```

Comment:
Use these roles for both 353 and 360
APPLICANTSOLICITORONESPEC
RESPONDENTSOLICITORONESPEC